frontend/appflowy_flutter/packages/flowy_infra/lib/image.dart

```diff
@@ -12,16 +11,11 @@ class FlowySvg extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (size != null) {
-      return SizedBox.fromSize(
-        size: size,
-        child: SvgPicture.asset('assets/images/$name.svg',
-            color: Theme.of(context).iconTheme.color),
-      );
-    } else {
-      return SvgPicture.asset('assets/images/$name.svg',
-          color: Theme.of(context).iconTheme.color);
-    }
+    return svgWidget(
```

Comment:
code for `svgWidget()`

```dart
Widget svgWidget(String name, {Size? size, Color? color}) {
  if (size != null) {
    return SizedBox.fromSize(
      size: size,
      child: SvgPicture.asset(
        'assets/images/$name.svg',
        colorFilter:
            color != null ? ColorFilter.mode(color, BlendMode.srcIn) : null,
      ),
    );
  } else {
    return SvgPicture.asset(
      'assets/images/$name.svg',
      colorFilter:
          color != null ? ColorFilter.mode(color, BlendMode.srcIn) : null,
    );
  }
}
```

Aside `svgWidget` should be a `Widget` and not a function that returns a `Widget`.